Makefile

```diff
@@ -3,14 +3,21 @@ include ./scripts/init.mk
 # This file contains hooks into the project configuration, test and build cycle
 # as automated steps to be executed on a workstation and in the CI/CD pipeline.
 
+install: # Install project dependencies
+	npm install
+
+build: # Build project
+	npm run build && npm run package
```

Comment:
Could we break this line down to two to improve readability? 

```
npm run build
npm run package
```

Reply:
Updated